var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -63,14 +71,15 @@ class PyPytest(PythonPackage):
     depends_on('py-py@1.4.33:', when='@3.1.2:3.2.3,3.2.5:3.2', type=('build', 'run'))
     depends_on('py-py@1.4.33:1.4', when='@3.2.4', type=('build', 'run'))
     depends_on('py-py@1.4.29:', when='@:3.1.1', type=('build', 'run'))
-    depends_on('py-toml', when='@6.0:', type=('build', 'run'))
+    depends_on('py-tomli', when='@7.0:', type=('build', 'run'))
     depends_on('py-atomicwrites@1.0:', when='@5.3: platform=windows', type=('build', 'run'))
     depends_on('py-atomicwrites@1.0:', when='@3.6:5.2', type=('build', 'run'))
     depends_on('py-colorama', when='platform=windows', type=('build', 'run'))
     depends_on('py-importlib-metadata@0.12:', when='@4.6:5.0', type=('build', 'run'))
     depends_on('py-importlib-metadata@0.12:', when='@5.1: ^python@:3.7', type=('build', 'run'))
 
     # Historic dependencies
+    depends_on('py-toml', when='@6.0:6.2.5', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-toml', when='@6', type=('build', 'run'))
```
In case they have a new 6.2.6 release